src/saveToCase.js

```diff
@@ -0,0 +1,475 @@
+                        "case_id",
+                    ],
+                },
+                {
+                    slug: "create",
+                    displayName: "Create",
+                    properties: [
+                        "use_create",
+                        "create_property",
+                    ],
+                    isCollapsed: function (mug) {
+                        return !createsCase(mug);
+                    },
+                },
+                {
+                    slug: "update",
+                    displayName: "Update",
+                    properties: [
+                        "use_update",
+                        "update_property",
+                    ],
+                    isCollapsed: function (mug) {
+                        return !updatesCase(mug);
+                    },
+                },
+                {
+                    slug: "close",
+                    displayName: "Close",
+                    properties: [
+                        "use_close",
+                        "close_condition",
+                    ],
+                    isCollapsed: function (mug) {
+                        return !closesCase(mug);
+                    },
+                }
+            ]
+        };
+
+    $.vellum.plugin("saveToCase", {}, {
+        getAdvancedQuestions: function () {
+            return this.__callOld().concat(["SaveToCase"]);
+        },
+        handleMugParseFinish: function (mug) {
+            this.__callOld();
+            // cases that are created use a setvalue for case_id
+            if (!createsCase(mug)) {
+                return;
+            }
+            var values = _.object(_.map(mug.form.getSetValues(), function (value) {
+                    return [value.ref, value];
+                }));
+
+            _.each(values, function(value) {
+                if (/case\/@case_id$/.test(value.ref)) {
+                    mug.p.case_id = value.value;
+                }
+            });
+        },
+        getMugTypes: function () {
+            var types = this.__callOld();
+            types.normal.SaveToCase = util.extend(
+                mugs.defaultOptions, saveToCaseMugOptions);
+            return types;
+        },
+        getSections: function (mug) {
+            if (sectionData.hasOwnProperty(mug.__className)) {
+                return _.map(sectionData[mug.__className], function (section) {
+                    var tmpSection = _.clone(section);
+                    if (_.isFunction(tmpSection.isCollapsed)) {
+                        tmpSection.isCollapsed = tmpSection.isCollapsed(mug);
+                    }
+                    return tmpSection;
+                });
+            }
+            return this.__callOld();
+        },
+        parseBindElement: function (form, el, path) {
+            var mug = form.getMugByPath(path);
+            if (!mug) {
+                var basePath = path.replace(/\/case\/[\w_]*\/[\w_]*$/, "");
```

Comment:
The [`\w` character class](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp#character-classes) includes `_`, btw.